packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.js

```diff
@@ -32,12 +32,43 @@ export default class QuanticFacet extends LightningElement {
   facet;
   /** @type {import("coveo").Unsubscribe} */
   unsubscribe;
+  /** @type {boolean} */
+  isCollapsed = false;
+  /** @type {string} */
+  facetIcon = 'utility:dash';
+  /** @type {HTMLInputElement} */
+  input;
+  /** @type {number} */
+  numberOfValues = 8;
+  /** @type  {import("coveo").FacetSortCriterion}*/
+  sortCriterion = 'automatic';
+  /** @type {boolean} */
+  isFacetSearchActive = false;
 
   labels = {
     showMore,
     showLess,
     showMoreFacetValues,
     showLessFacetValues,
+    clear,
+    search,
+    moreMatchesFor,
+    noMatchesFor,
+  };
+
+  /**
+   * @param {import("coveo").SearchEngine} engine
+   */
+  @api
+  initialize(engine) {
+    const options = {
```

Comment:
You prefer the extra variable definition?

Reply:
I thought it made it easier to read, but since there aren't many options I'm good with either way

Reply:
I was just making sure it was a conscious decision, no problem on my part.